modules/nf-core/summarizedexperiment/summarizedexperiment/main.nf

```diff
@@ -30,7 +30,7 @@ process SUMMARIZEDEXPERIMENT_SUMMARIZEDEXPERIMENT {
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
-        r: \$( R --version | sed '1!d; s/.*version //; s/ .*//' )
```

Comment:
Was `R --version` producing a timestamp of some sort?

Reply:
It's just that the R version drifts over time in Conda, while it doesn't in the biocontainers. So I took out the R version and replaced it with the package version. Not sure why I was using the R version in the first place, must have copied it from somewhere.